helm/chart/maesh/templates/_helpers.tpl

```diff
@@ -7,10 +7,19 @@ Define the Chart version Label
 {{- define "maesh.chartLabel" -}}
     {{- printf "%s-%s" .Chart.Name .Chart.Version -}}
 {{- end -}}
-
 {{/*
 Define the templated image with tag
 */}}
 {{- define "maesh.controllerImage" -}}
     {{- printf "%s:%s" .Values.controller.image.name ( .Values.controller.image.tag | default .Chart.AppVersion ) -}}
 {{- end -}}
+{{/*
+Define the ignoreNamespaces List
+*/}}
+{{- define "maesh.controllerIgnoreNamespaces" -}}
+    --ignoreNamespaces=
+    {{- range $idx, $ns := .Values.controller.ignoreNamespaces }}
```

Comment:
What happens if this is empty?
does it just result in:
```
--ignoreNamespaces=
```
?

And does that break anything?

Reply:
Yes it will create an empty --ignoreNamespaces, but it doesn't break anything. However, I added a sanity check. Thanks!